File: android/app/src/main/java/com/sumitrack/android/domain/models/Sale.kt
package com.sumitrack.android.domain.models

import java.math.BigDecimal
import java.time.Instant

data class Sale(
    val id: String,
    val fkTenant: String,
    val fkClient: String,
    val folio: String,
    val total: BigDecimal,
    val status: SaleStatus,
    val createdAt: Instant,
    val updatedAt: Instant,
    val syncStatus: SyncStatus,
)

File: backend/src/Sumitrack.Api/Models/Entities/Sale.cs
namespace Sumitrack.Api.Models.Entities;

public class Sale
{
    public Guid Id { get; set; }
    public Guid FkTenant { get; set; }
    public Guid FkClient { get; set; }
    public string Folio { get; set; } = string.Empty;
    public decimal Total { get; set; }
    public decimal Subtotal { get; set; }
    public decimal Tax { get; set; }
    public string Status { get; set; } = "pending";
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    public string SyncStatus { get; set; } = "synced";
}
